paz/datasets/vvad_lrs3.py

```diff
@@ -0,0 +1,209 @@
+import os
+import h5py
+import random
+import numpy as np
+import math
+from typing import Literal, get_args
+
+from .utils import get_class_names
+from ..abstract import Generator
+
+Reduction_Method = Literal["cut", "reduce"]
+
+
+class VvadLrs3Dataset(Generator):
```

Comment:
I would move this file outside to the examples directory. We can then later work to integrate it. Would that be OK?